index.js

```diff
@@ -4,7 +4,14 @@ var semver = require('semver-regex');
 var ZSchema = require('z-schema');
 var jsf = require('json-schema-faker');
 
-var schemas = require('./schemas');
+var originalSchemas = require('./schemas')
+
+function deepCopy (json) { return JSON.parse(JSON.stringify(json)) }
+// this is an easy way to get immutable schemas and eschew
+// zaggino/z-schema#160 . I think that performance is not a problem
+// for us here
+function getSchemas () { return deepCopy(originalSchemas) }
+var schemas = getSchemas()
```

Comment:
`schemas` is more performant, `getSchemas` is immutable. Performance doesn't matter so much so we could also use `getSchemas` everywhere, but i find it a bit confusing, so when we are not afraid of having our data mutated in place we can simply use `schemas`